### PR TITLE
Add more info to the README

### PR DESCRIPTION
**Pull request by mattlord**
Signed-off-by: Matt Lord <mattalord@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # vitess-bot
 
-Is the bot that automates redundant tasks in `vitessio/vitess`. The following is automated:
+This bot automates redundant tasks in the [`vitessio/vitess`](https://github.com/vitessio/vitess) git repo.
 
-- Comment a review checklist on any Pull Request that is ready for review.
-- Backport a Pull Request to release branches.
\ No newline at end of file
+It currently currently automates the following tasks:
```
Comment (deepthi) [inline README.md:5]:
```suggestion
It currently automates the following tasks:
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
 # vitess-bot
 
-Is the bot that automates redundant tasks in `vitessio/vitess`. The following is automated:
+This bot automates redundant tasks in the [`vitessio/vitess`](https://github.com/vitessio/vitess) git repo.
```
Comment (deepthi) [inline README.md:3]:
```suggestion
This bot automates some tasks in the [`vitessio/vitess`](https://github.com/vitessio/vitess) git repo.
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
 
-- Comment a review checklist on any Pull Request that is ready for review.
-- Backport a Pull Request to release branches.
\ No newline at end of file
+It currently currently automates the following tasks:
+- Adds a review checklist comment on any Pull Request that is ready for review
+- Creates backports against previous release branches for a Pull Request based on the original PR's backport labels
+  - This will fail if there are any conflicts or other issues which require manual intervention
```
Comment (deepthi) [inline README.md:8]: It also does forward ports. We should document that too.
And list the exact label values that trigger a back port or forward port

**Review by deepthi — APPROVED** [review]
Approving so that you (or Florent) can merge as soon as the edits are complete.

File: README.md (hunk 3 above)
Comment (mattlord) [reply README.md]: Thanks! I've addressed your great ideas in https://github.com/vitessio/vitess-bot/pull/14/commits/df95a02521747b3a69718522ad552b52a5e6ca3f and https://github.com/vitessio/vitess-bot/pull/14/commits/90badc1df5ddb376dc9b37036ef54204b059a6a9

We can improve upon it more over time.

**Pull request merged**